src/app/main/component/auth/components/restore-password/restore-password.component.html

```diff
@@ -3,44 +3,44 @@ <h1>{{ 'user.auth.forgot-password.title' | translate }}</h1>
   <h2>{{ 'user.auth.forgot-password.fill-form' | translate }}</h2>
   <form class="restore-password-form" (ngSubmit)="sentEmail()" [formGroup]="restorePasswordForm">
     <label for="email">{{ 'user.auth.sign-in.email' | translate }}</label>
-    <input
-      id="email"
-      type="email"
-      placeholder="example@email.com"
-      formControlName="email"
-      name="email"
-      required
-      (input)="configDefaultErrorMessage()"
-      [ngClass]="
-        (emailField.invalid && emailField.touched) || backEndError || emailErrorMessageBackEnd
-          ? 'alert-email-validation'
-          : 'successful-email-validation'
-      "
-    />
-    <div class="validation-email-error" *ngIf="emailField.invalid && emailField.touched">
-      <app-error [formElement]="emailField" [emailFieldValue]="emailFieldValue" [controlName]="'email'"></app-error>
+    <div class="input-wrapper">
+      <input
+        id="email"
+        type="email"
+        placeholder="example@email.com"
+        formControlName="email"
+        name="email"
+        required
+        (input)="configDefaultErrorMessage()"
+        [ngClass]="
+          (emailField.invalid && emailField.touched) || backEndError || emailErrorMessageBackEnd
+            ? 'alert-email-validation'
+            : 'successful-email-validation'
+        "
```

Comment:
You can make this code pretier )),  if you implements class check as method  classCheck(){
(emailField.invalid && emailField.touched) || backEndError || emailErrorMessageBackEnd
            ? 'alert-email-validation'
            : 'successful-email-validation'}